screens/mycart.html

```diff
@@ -0,0 +1,145 @@
+      </div>
+      <div class="flex-row header-right">
+        <button class="btn primary-btn">LOGIN</button>
+        <a href="../screens/wishlist.html">
+          <div class="relative-container wishl-icon">
+            <i class="material-icons fav-icon">
+              favorite_border
+              <div class="badge top-right lrg red-clr flex-row center-it">
+                3
+              </div></i
+            >
+          </div></a
+        >
+        <a href="#" class="cart">
+          <div class="flex-row relative-container">
+            CART
+            <i class="material-icons"> shopping_cart </i>
+            <div class="badge top-right lrg red-clr flex-row center-it">2</div>
+          </div></a
+        >
+        <span class="material-icons"> logout </span>
+      </div>
+    </header>
+    <!--  -->
+    <main class="cart-container">
+      <h1 class="cart-heading">My Cart(2)</h1>
+      <section class="flex-row center-it gap-btwn">
+        <div class="flex-col">
+          <div class="card card-horiz gap-btwn">
+            <img class="card-img" src="../assets/products/p1.jpg" alt="demo" />
+            <div class="card-info">
+              <h5 class="card-title">God of War</h5>
+              <p class="description">PEGI Rating: Ages 16 and Above</p>
+              <div class="card-rating">
+                4.5 <span class="material-icons star-icon"> star </span>
+              </div>
+
+              <span class="card-price"
+                ><sup>₹</sup>2,970<s>5,499</s
+                ><span class="card-discount">Save 2,529</span></span
+              >
+              <div class="flex-row center-it gap-btwn">
+                <p>Quantity:</p>
+                <button class="cart-quan-btn">+</button>
+                <div class="cart-quant-cnt">2</div>
+                <button class="cart-quan-btn">-</button>
+              </div>
+              <button class="card-btn primary-btn">BUY NOW</button>
+              <button class="card-btn card-icon-btn">
+                <span class="material-icons icon"> shopping_cart </span>
+                Add to Cart
+              </button>
+            </div>
+          </div>
+          <div class="card card-horiz gap-btwn">
+            <img class="card-img" src="../assets/products/p6.jpg" alt="demo" />
+            <div class="card-info">
+              <h5 class="card-title">God of War</h5>
+              <p class="description">PEGI Rating: Ages 16 and Above</p>
+              <div class="card-rating">
+                4.5 <span class="material-icons star-icon"> star </span>
+              </div>
+
+              <span class="card-price"
+                ><sup>₹</sup>2,970<s>5,499</s
+                ><span class="card-discount">Save 2,529</span></span
+              >
+              <div class="flex-row center-it gap-btwn">
+                <p>Quantity:</p>
+                <button class="cart-quan-btn">+</button>
+                <div class="cart-quant-cnt">2</div>
```

Comment:
better to keep consistent in naming to classes, here it's -quan and -quant ,is used.

Reply:
I think I forgot thank you to point out